File: trunk/src/DemoVB/Item.vb
Public Class Item

    Private _title As String
    Private _description As String
    Private _pubDate As DateTime

    Public Property Title() As String
        Get
            Return _title
        End Get
        Set(ByVal value As String)
            _title = value
        End Set
    End Property

    Public Property Description() As String
        Get
            Return _description
        End Get
        Set(ByVal value As String)
            _description = value
        End Set
    End Property

    Public Property PubDate() As DateTime
        Get
            Return _pubDate
        End Get
        Set(ByVal value As DateTime)
            _pubDate = value
        End Set
    End Property

End Class

File: trunk/src/DemoCS/Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Demo
{
    /// <summary>
    /// An RSS feed item.
    /// </summary>
    class Item
    {
        private string _title;
        private string _description;
        private DateTime? _pubDate;

        public string Title
        {
            get { return _title; }
            set { _title = value; }
        }
        
        public string Description
        {
            get { return _description; }
            set { _description = value; }
        }
        
        public DateTime? PubDate
        {
            get { return _pubDate; }
            set { _pubDate = value; }
        }

    }
}
